source/includes/fact-valid-namespace-remap.rst

```diff
@@ -0,0 +1,35 @@
+
+The following restrictions apply to namespace remapping:
+
+- Namespace remapping doesn't support changes to collection names.
+
+- Namespace remapping doesn't permit writing to the ``system``, ``config``, ``admin``,
+  or ``local`` databases, or writing to internal databases used by ``mongosync``.
+
+- The database name on the destination cluster must be valid under Windows
+  restrictions.
+
+  For more information, see :limit:`Restrictions on Database Names for Windows`.
+
+- Remapped database names on the destination cluster cannot differ only in case.
+
+- The remap cannot produce namespace conflicts on the destination cluster.
+
+  For example:
+
+  .. code-block:: javascript
+     :copyable: false
+
+     "namespaceRemap": [
+       {
+          "from": { "database": "us-west" },
+          "to": {"database": "us-accounts" }
+       },
+       {
+          "from": { "database": "us-south" },
+          "to": { "database": "us-accounts" }
+       }
+     ]
+
+  This remap would generate an namespace error if each database on the source cluster 
+  contains a ``texas`` collection.
```

Comment:
We unfortunately won't be able to catch and error for this specific restriction. Can we instead say something like "mongosync may exhibit undefined behavior / corrupt data" if this happens?

Reply:
Fixed